File: Service/Implement/ATTPTiepNhanProductGroupsService.cs
namespace Service.Implement
{
    public class ATTPTiepNhanProductGroupsService : BaseService<ATTPTiepNhanProductGroups, IATTPTiepNhanProductGroupsRepository>
    , IATTPTiepNhanProductGroupsService
    {
    private readonly IATTPTiepNhanProductGroupsRepository _ATTPTiepNhanProductGroupsRepository;
    public ATTPTiepNhanProductGroupsService(IATTPTiepNhanProductGroupsRepository ATTPTiepNhanProductGroupsRepository) : base(ATTPTiepNhanProductGroupsRepository)
    {
    _ATTPTiepNhanProductGroupsRepository = ATTPTiepNhanProductGroupsRepository;
    }
    }
    }

File: CMS/src/app/shared/ATTPTiepNhanProductGroups.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { ATTPTiepNhanProductGroups } from 'src/app/shared/ATTPTiepNhanProductGroups.model';
import { BaseService } from './Base.service';
@Injectable({
  providedIn: 'root'
})
export class ATTPTiepNhanProductGroupsService extends BaseService {
  constructor(public httpClient: HttpClient) {
    super(httpClient);
    this.Controller = "ATTPTiepNhanProductGroups";
  }
}

